### PR TITLE
Fix version numbers

### PR DESCRIPTION
**Pull request by faustinoaq**
### Description of the Change

Fix version numbers on `shard.yml` and updates dependencies

https://shards.rocks/github/amberframework/amber

screenshot_20171021_090104

### Alternate Designs

No

### Benefits

No

### Possible Drawbacks

No

### CODE REVIEW

**Comment by faustinoaq** [comment]
@eliasjpr Travis is failing

Maybe `github: mosop/cli` had been changed on latest version `v0.6.10`

**Comment by faustinoaq** [comment]
This PR also removes forks and fixes definitely #261 started by @elorest on #291

**Comment by faustinoaq** [comment]
Finally works! :)

**Comment by faustinoaq** [comment]
@drujensen @fridgerator I think this PR is ready, WDYT? 😅

File: src/amber/cli/commands/database.cr [hunk 1]
```diff
@@ -19,6 +19,8 @@ module Amber::CLI
       end
 
       def run
+        migrations_path, migrations_table_suffix = Micrate::Cli.parse_command_arguments
```
Comment (elorest) [inline src/amber/cli/commands/database.cr]: Why was this necessary. Looks like we're just passing commands back into Micrate that it gave us in the same scope.

File: src/amber/cli/commands/migrate.cr [hunk 2]
```diff
@@ -20,25 +20,26 @@ module Amber::CLI
       end
 
       def run
+        migrations_path, migrations_table_suffix = Micrate::Cli.parse_command_arguments
```
Comment (elorest) [inline src/amber/cli/commands/migrate.cr]: Ditto.

File: src/amber/cli/commands/sentry_command.cr [hunk 3]
```diff
@@ -0,0 +1,103 @@
+require "yaml"
+require "cli"
+
+require "sentry"
+
+module Sentry
```
Comment (elorest) [inline src/amber/cli/commands/sentry_command.cr]: Is this meant to replace `watch`

File: src/amber/cli/commands/database.cr (hunk 1 above)
Comment (faustinoaq) [reply src/amber/cli/commands/database.cr]: I just fixed it to work with latest version of micrate shard see: https://github.com/juanedi/micrate/commit/af301fcec0c42ee5872fe0d783fbf91a762a11b9

File: src/amber/cli/commands/migrate.cr (hunk 2 above)
Comment (faustinoaq) [reply src/amber/cli/commands/migrate.cr]: I just fixed it to work with latest version of micrate shard see: https://github.com/juanedi/micrate/commit/af301fcec0c42ee5872fe0d783fbf91a762a11b9

File: src/amber/cli/commands/sentry_command.cr (hunk 3 above)
Comment (faustinoaq) [reply src/amber/cli/commands/sentry_command.cr]: No, It's just a helper to avoid using https://github.com/TechMagister/sentry/tree/change_cli

File: src/amber/cli/commands/database.cr (hunk 1 above)
Comment (faustinoaq) [reply src/amber/cli/commands/database.cr]: I reverted this changes, please check this PR again 😄

File: src/amber/cli/commands/migrate.cr (hunk 2 above)
Comment (faustinoaq) [reply src/amber/cli/commands/migrate.cr]: I reverted these changes, please check this PR again 😄

File: src/amber/cli/commands/sentry_command.cr (hunk 3 above)
Comment (faustinoaq) [reply src/amber/cli/commands/sentry_command.cr]: I moved this file and rename to more explicit name

**Comment by faustinoaq** [comment]
@elorest This PR is removing fork as shards successfully, Also I reverted micrate changes, (using `0.3.0`)

**Comment by faustinoaq** [comment]
Please squash before merge, I did more than 20 commits because I was using github from web 😅

**Comment by veelenga** [comment]
Hey guys, can this be merged/released before crystal 0.24.0 ?

**Comment by faustinoaq** [comment]
For some reason Amber Task merged here 👉 https://github.com/amberframework/amber/pull/318 needs a ols version of `mosop/cli` 😅 

However, this PR is working now 👍 We can figure out about updating new dependencies on future

I will open a new issue about this 😉
